cs3/app/provider/v1beta1/resources.proto

```diff
@@ -44,6 +44,9 @@ message OpenInAppURL {
   // OPTIONAL.
   // The headers to be added to the request.
   map<string, string> headers = 4;
+  // REQUIRED.
+  // Whether the target for the app URL is an iframe or a new page.
+  Target target = 5;
```

Comment:
I'm not sure how `target` fits in here. Eg. doing a POST to a new tab is technically not possible.

ownCloud Web has a option `WEB_OPTION_OPEN_APPS_IN_TAB` -> `Configures whether apps and extensions should generally open in a new tab. It seems to fulfill the same purpose? In the future, this could also be a per user setting.

Reply:
Yes, with this not all combinations are legal, but the scope is for *certain* applications to go to a new tab/page as opposed to an iframe. We are trying to integrate Overleaf, but the integration would work quite differently than WOPI-based apps, therefore it makes sense (and it is necessary for CSP reasons) to have the user go directly to the app's URL, rather than embedding it in an iframe.

Reply:
Ah, ok this is currently not configure for oC Web. Because even if one configures Web to open everything in a new tab, it will open the app inside a iframe via the GET / POSt request.

@micbar this is highly interesting for future integrations